app/src/main/java/com/termux/app/ExtraKeysView.java

```diff
@@ -59,6 +57,12 @@ static void sendKey(View view, String keyName) {
             case "PGDN":
```

Comment:
Add aliases, PAGE_DOWN, PAGEDOWN etc. People are likely to have that in their termux.properties.
Same for INSERT, DELETE, BACKSPACE, UP DOWN LEFT RIGHT etc. A short name that is the preffered one, and longer name explicit.